Misc/NEWS.d/next/C API/2022-09-15-15-21-34.gh-issue-96803.ynBKIS.rst

```diff
@@ -0,0 +1,9 @@
+Expose C-API functions to get the code object, lasti and line number from
```

Comment:
Not sure if this needs a news entry since the functions are exposed technically on a private header.